packages/grid/scripts/backup_node_credentials.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+ROOT_DATA_PATH="$HOME/.syft/data"
+
+calculate_checksum() {
+    # Calculate the checksum of a file
+    local file="$1"
+    local checksum
+    checksum=$(sha256sum "$file" | awk '{print $1}')
+    echo "$checksum"
+}
+
+docker_cp() {
+    # Exit in case of error
+    set -e
+
+    # Define the container name pattern
+    CONTAINER_NAME_PATTERN="backend-1"
+
+    # Get a list of container names that match the pattern
+    CONTAINER_NAMES=$(docker ps --filter "name=$CONTAINER_NAME_PATTERN" --format "{{.Names}}")
+
+    if [ -z "$CONTAINER_NAMES" ]; then
+        echo "No containers found with the name pattern: $CONTAINER_NAME_PATTERN"
+        exit 1
+    fi
+
+    for CONTAINER_NAME in $CONTAINER_NAMES; do
+        # Define the source path of the credentials.json file in the container
+        SOURCE_PATH="/storage/credentials.json"
```

Comment:
Hi @PeterChung241 This merged change https://github.com/OpenMined/PySyft/pull/8179 tweaked the path where syft stores data & credentials.
* `/root/data/creds/credentials.json`
* `/root/data/db` is where sqlite db is stored

With this you now have the option of backing up the whole `/root/db`.

Also, once we move from root user `/root/` to a rootless user `/root/syftuser` these paths will change again. So I'll recommend we move `SOURCE_PATH` `FILE_PATH_IN_POD`, etc to a common variable at the start of that scripts. that way it'll be easier to track and change backup paths in the future.

Reply:
Thanks Yash, changes and suggestion have been incorporated.